lm75bd/lm75bd.c

```diff
@@ -26,9 +26,43 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
   return ERR_CODE_SUCCESS;
 }
 
-error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
+error_code_t readTempLM75BD(uint8_t devAddr, float *temp)
+{
   /* Implement this driver function */
-  
+  if (!temp)
+  {
+    return ERR_CODE_INVALID_ARG;
+  }
+
+  uint8_t tempPointerRegister = 0x00;
+  uint8_t buff[2] = {0};
```

Comment:
I would declare this variable after i2cSendTo since you don't need it right now so it's a bit confusing to be declaring it here, especially since if i2cSendTo fails there is no need for this variable

Reply:
I don't think this was addressed